becquerel/tools/materials_compendium.py

```diff
@@ -60,6 +60,12 @@ def fetch_compendium_data():
             data = json.load(f)
 
     # extract relevant data
+    if isinstance(data, list):
+        print("Pre-March 2022 JSON detected")
+    elif isinstance(data, dict):
+        print("Post-March 2022 JSON detected")
+        print(f"Compendium data - site version: {data['siteVersion']}")
+        data = data["data"]
```

Comment:
Can you add an `else` with a suitable error here, just in case?

Reply:
Good call.